packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -278,12 +279,8 @@ export class EngineReportingAgent<TContext = any> {
     handleLegacyOptions(this.options);
   }
 
-  public newExtension(schemaHash: SchemaHash): EngineReportingExtension<TContext> {
-    return new EngineReportingExtension<TContext>(
-      this.options,
-      this.addTrace.bind(this),
-      schemaHash,
```

Comment:
`schemaHash` is now received on the `requestContext` of the `requestDidStart` and the plugin is no longer instantiated entirely anew for each request but rather once per server startup.  Thus, this parameter is no longer necessary.